jenkins/osa_onmetal.functions.groovy

```diff
@@ -374,7 +374,7 @@ def setup_parse_persistent_resources(){
     
     sh """
     ssh -o StrictHostKeyChecking=no root@${host_ip} '''
-    git clone https://github.com/osic/persistent-resources-tests-parse.git
+    git clone https://github.com/osic/persistent-resources-tests-parse.git /root/persistent-resources-tests-parse
     pip install -e /root/persistent-resources-tests-parse
```

Comment:
For some reason the package will not install if it's not install in development mode.. I researched about it and the solutions presented to solved problem my package met the requirements already but it still didn't install correctly.